packages/tupaia-web/src/utils/transformDownloadLink.ts

```diff
@@ -10,5 +10,6 @@ import { API_URL } from '../api';
 export const transformDownloadLink = (resourceUrl: string) => {
   const baseUrl = API_URL;
   if (resourceUrl.includes('http')) return resourceUrl;
-  return `${baseUrl}${resourceUrl}`;
+  if (baseUrl.endsWith('/')) return `${baseUrl}${resourceUrl}`; // remove trailing slash from baseUrl if it exists
```

Comment:
I think that [trim()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim) would do the trick here, right?

Reply:
That would only work with whitespace right?

Reply:
Oh yeah sorry - you are totally correct - I didn't read it correctly.